source/crud/crud.md

```diff
@@ -1070,6 +1069,18 @@ class UpdateOptions {
    * and providing one will result in a server-side error.
    */
   comment: Optional<any>;
+
+  /**
+   * Specify which document the operation updates if the query matches multiple
+   * documents. The first document matched by the sort order will be updated.
+   *
+   * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+   * This option is only supported by servers >= 8.0. Older servers will report an error for using this option.
+   * The driver MUST raise an error if the caller explicitly provides a value with updateMany().
```

Comment:
This seems like it'd be an annoyance for drivers that share a common, typed UpdateOptions structure for both `updateOne()` and updateMany()`. I have a vague recollection of similar situations coming up in the past, perhaps most recently during the new `bulkWrite` spec design.

@jyemin: Can you chime in with a Java perspective?

Reply:
Can we instead rely on the server to error? IE the "depend on server to return errors" mantra: https://github.com/mongodb/specifications/blob/master/source/driver-mantras.md#where-possible-depend-on-server-to-return-errors